code/_onclick/ai.dm

```diff
@@ -63,6 +66,10 @@
 		waypoint_mode = 0
 		return
 
+	if(controlled_mech) //Are we piloting a mech? Placed here so the modifiers are not overridden.
```

Comment:
We can do that with call()() but call()() is a weird type unsafe beast.

```
AiClickOn()
    call(whichAIClick)()


whichAIClick = /mob/living/silicon/ai/MechCleck()
```
etc.